test/ILLink.RoslynAnalyzer.Tests/RequiresUnreferencedCodeAnalyzerTests.cs

```diff
@@ -234,7 +240,12 @@ Action M2()
 					// /0/Test0.cs(12,28): warning IL2026: Using method 'C.M1()' which has 'RequiresUnreferencedCodeAttribute' can break functionality when trimming application code. message.
 					VerifyCS.Diagnostic().WithSpan(12, 28, 12, 32).WithArguments("C.M1()", "message", "")
 				},
-				fixedExpected: Array.Empty<DiagnosticResult> ());
+				fixedExpected: Array.Empty<DiagnosticResult> (),
+				// This test needs to declare the number of iterations it will run, by default the number of iterations is calculated counting the number of DiagnosticResults
```

Comment:
Note sure I understand the comment, lemme see if I can rephrase:

The default iterations for the codefix is the number of diagnostics (1 in this case),
but since the fixer introduces a new diagnostic in the first iteration, it needs to run
twice, so we set the number of iterations to 2.

Reply:
I like your comment, I will change the comment